mathics/doc/doc_entries.py

```diff
@@ -346,12 +346,66 @@ def strip_sentinal(line: str):
     def __str__(self) -> str:
         return self.test
 
+    def compare(self, result: Optional[str], out: Optional[tuple] = tuple()) -> bool:
+        """
+        Performs a doctest comparison between ``result`` and ``wanted`` and returns
+        True if the test should be considered a success.
+        """
+        return self.compare_result(result) and self.compare_out(out)
+
+    def compare_result(self, result: Optional[str]):
+        """Compare a result with the expected result"""
+        wanted = self.result
+        # Check result
+        if wanted in ("...", result):
+            return True
+
+        if result is None or wanted is None:
+            return False
+        result_list = result.splitlines()
+        wanted_list = wanted.splitlines()
+        if result_list == [] and wanted_list == ["#<--#"]:
+            return True
+
+        if len(result_list) != len(wanted_list):
+            return False
+
+        for res, want in zip(result_list, wanted_list):
+            wanted_re = re.escape(want.strip())
+            wanted_re = wanted_re.replace("\\.\\.\\.", ".*?")
+            wanted_re = f"^{wanted_re}$"
+            if not re.match(wanted_re, res.strip()):
+                return False
+        return True
+
+    def compare_out(self, outs: tuple = tuple()) -> bool:
+        """Compare outs of the test with the expeted outs"""
```

Comment:
expeted -> expected. 

Also "outs" is jargon. It refers to the system messages (as opposed to computation or result output. It would be nice to clarify here, and above where "results" is described as computation output not system messages.